servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Single.java

```diff
@@ -670,6 +780,45 @@ public final Completable flatMapCompletable(Function<? super T, ? extends Comple
      * @return The new {@link Single}.
      */
     public final Single<T> beforeFinally(Runnable doFinally) {
+        return beforeFinally(TerminalSignalConsumer.from(doFinally));
```

Comment:
if we still expose the `Runnable` variants can we hide `TerminalSignalConsumer.from` for now?